src/features/LandingPage/QuickStartSection/index.tsx

```diff
@@ -47,7 +47,8 @@ const QuickStartSection = () => {
           <div className={styles.copyCodeBlock}>
             <pre id="dockerSnippet" className={styles.dockerSnippet}>
               docker pull openfga/openfga && \<br />
-              docker run -p 8080:8080 openfga/openfga run
+              docker run -p 8080:8080 -p 8081:8081 \<br />
```

Comment:
Why don't we just put these each on their own line? I think it would make the command look better, rather than mixing and matching.

For example,

```
docker run \
    -p 8080:8080 \
    -p 8081:8081 \
    -p 3000:3000 \
    openfga/openfga run
```

WDYT?